src/preact/base-element.js

```diff
@@ -166,13 +277,16 @@ export class PreactBaseElement extends BaseElement {
     /** @private {boolean} */
     this.resetLoading_ = false;
 
-    /** @private {?API_TYPE} */
+    /** @private {API_TYPE|null} */
```

Comment:
This annotation doesn't provide a type, as you discovered 🙃

Reply:
This one actually does!

Reply:
Wow that's surprising, but cool!